Documents/Technical Specifications/Technical.md

```diff
@@ -0,0 +1,494 @@
+It's already made and we can use it as a base for our game.
+All materials and textures are already made and applied to the asset.
+
+## Character
+
+### Sube
+
+Sube is the main character and is a sugar cube.
+For the MVP, Sube will be a simple cube with a material of sugar applied to it.
+For the final version, Sube will be a 3D model of a sugar cube with a material of sugar applied to it. It will have to be designed by us on Blender.
+
+### NPC
+
+Npc are going to be other sugar cubes that will be used to interact with the player.
+It may be used to give the player a hint or lore about the game.
+For the MVP, Npc will be a simple cube with a material of sugar applied to it.
+For the final version, Npc will be a 3D model of a sugar cube with a material of sugar applied to it. It will have to be designed by us on Blender.
+
+## Plugin
+
+Gravity change plugin is a plugin that allows us to change the gravity of the player. It is available [here](https://www.unrealengine.com/marketplace/en-US/product/directional-planet-gravity). It is free to use for non-commercial projects. It will be used for the sticky functionnality.
+
+# Gameplay
+
+## Objective
+
+The objective of the game is to reach the cup of coffee. The cup of coffee is the end of the game.
+The game will be separated in levels. Each level will move the cup of coffee further away from the player.
+
+## Camera
+
+## Mechanics
+
+Each mechanic will be explained in details here.
+
+### Movement
+
+The player will be able to move on 2 axis (X / Y by default).
+The player will be able to move at a constant speed.
+The player will be able to move in 4 directions (Up / Down / Left / Right).
+Each direction will be mapped to a key.
+
+- Forward: Z
+- Backward: S
+- Left: Q
+- Right: D
+
+Each key when pressed will trigger a movement in the corresponding direction.
+The will also trigger the associated animation for the cube to move.
+While the cube is moving, any other interractions will be disabled to avoid unwanted behavior.
+
+### Obstacle
+
+Obstacles will be placed in the level to prevent the player from reaching the cup of coffee.
+Obstacles will be items integrated in the environment and might take any forms ( Wall, plate, etc. ).
+Each obstacle will have a collision box for the player to not glitch through it.
+Each obstacle will have a material to make it visible.
+Each obstacles placement and details will be define in the [Level section](#level).
+
+### Sticky
+
+Sticky will be a mechanic that will allow the player to stick to walls.
+It will be shown as a green particle effect on the wall as if the wall was dirty.
+It will activate when the player enter a special zone.
+And will stay activated as long as the player stay in that zone and follow the predefined path.
+The controls stay relative to the camera.
+It means old controls become as follow:
+
+- Forward: Z -> Forward
+- Backward: S -> Backward
+- Left: Q -> Down
+- Right: D -> Up
```

Comment:
```suggestion
It means old controls become as follow:

|          | Key | Alternative Key | Action   |
|----------|-----|-----------------|----------|
| Forward  | Z   | ↑               | Forward  |
| Backward | S   | ↓               | Backward |
| Left     | Q   | ←               | Down     |
| Right    | D   | →               | Up       |
```

Reply:
Kept the table part
But Arrow might be used in the future for something else

We might use arrows for the second cube. 
We might add arrow as Alternate Key if we don't introduce the split feature